spec/displaynames.html

```diff
@@ -245,28 +245,34 @@ <h1>Properties of Intl.DisplayNames Instances</h1>
   <emu-clause id="sec-intl-displaynames-abstracts">
     <h1>Abstract Operations for DisplayNames Objects</h1>
 
-    <emu-clause id="sec-canonicalcodefordisplaynames" aoid="CanonicalCodeForDisplayNames">
-      <h1>CanonicalCodeForDisplayNames ( _type_, _code_ )</h1>
-      <p>
-        The CanonicalCodeForDisplayNames abstract operation takes arguments _type_ (a String) and _code_ (a String). It verifies that the _code_ argument represents a well-formed code according to the _type_ argument and returns the case-regularized form of the _code_. The algorithm refers to <a href="https://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>. The following steps are taken:
```

Comment:
Why do you want to remove the part mentioned about UTS35 here?

Reply:
Because it is now handled by the auto-linked "language tag nonterminal" term.